ansible/docker_site.yml

```diff
@@ -12,11 +12,14 @@
     - {role: ci/servers/local/init, tags: ["local"]}
     - {role: ci/build/pr_only, tags: ["build_pr_only"]}
     - {role: ci/init/check_cache, when: run_build, tags: ["check_cache"]}
-    - {role: ci/init/install, when: run_install, tags: ["install"]}
-    - {role: ci/init/create_workspace, when: run_install, tags: ["create_workspace"]}
-    - {role: ci/update_dependencies, when: run_build, tags: ["update_dependencies", "build", "unit_tests",
-                                                             "code_coverage", "codecov_tool", "python_code_coverage",
-                                                             "cpp_code_coverage", "check_deb", "check_build"]}
+    - {role: ci/init/install, when: run_install, tags: ["build", "unit_tests", "code_coverage", "codecov_tool", "python_code_coverage",
+                                                        "cpp_code_coverage", "check_deb", "check_build", "python_code_style_check",
+                                                        "cpp_code_style_check", "code_style_check"]}
+    - {role: ci/init/create_workspace, when: run_install, tags: ["build", "unit_tests", "code_coverage", "codecov_tool", "python_code_coverage",
```

Comment:
roles install and create_workspace has the same dependencies. Basically they are needed for almoust any role. So I have removed their role tags and just added tags of all roles.